src/camera-helper.ts

```diff
@@ -30,15 +30,23 @@ if (constants.SUPPORTS_MEDIA_DEVICES) {
 }
 
 export default class CameraHelper {
+  flash: FillLightMode;
+  redEyeReduction: boolean;
+
   private stream: MediaStream | null;
   private track: MediaStreamTrack | null;
   private trackConstraints: MediaTrackSettings;
+  private photoCapabilities: PhotoCapabilities | null;
 
   constructor() {
     this.stream = null;
     this.track = null;
 
+    this.photoCapabilities = null;
     this.trackConstraints = {};
+
+    this.flash = 'off';
```

Comment:
`off` seems like an odd value. I believe it should be `''` in your case.

Reply:
This is what the *default* should be, and it needs to be a valid value so I can pass it as-is in the `PhotoSettings`.

If I made it nullable, I'd have to set it to some default anyway if/when the UI button is displayed, but I'd also have to sprinkle in null-checking throughout the rest of the code.